run-build-functions.sh

```diff
@@ -79,7 +81,7 @@ install_deps() {
 }
 
 run_yarn() {
-  yarn_version=$1
+  local yarn_version=$1
```

Comment:
This changes seems unrelated to pnpm, would it be possible to remove it?

Reply:
Sure